packages/templating/dynamic.html

```diff
@@ -17,6 +25,11 @@
 the template to render) and a `data` property, which can be falsey. -->
 <template name="__dynamicWithDataContext">
   {{#with chooseTemplate template}}
-    {{> .. ../data}}    {{!-- The .. is evaluated inside {{#with ../data}} --}}
+    {{#if Template.contentBlock}}
```

Comment:
I don't think this if statement actually does anything - if I recall correctly `contentBlock` is a function that's _always_ defined. Am I wrong?